src/application/components/EmptyMessage.tsx

```diff
@@ -0,0 +1,19 @@
+import { clsx } from "clsx";
+
+interface EmptyMessageProps {
+  className?: string;
+}
+
+export function EmptyMessage({ className }: EmptyMessageProps) {
+  return (
+    <div className={clsx(className, "w-[344px] text-center")}>
+      <h1 className="text-md font-semibold font-body text-heading dark:text-heading-dark">
+        👋 Welcome to Apollo Client Devtools
+      </h1>
+      <div className="[grid-area:content]">
```

Comment:
Does this make sense? The parent `div` is not a grid and doesn't seem to have `display: contents`

Reply:
Oops, I think this was left over from a previous refactor. Fixed! [4504499](https://github.com/apollographql/apollo-client-devtools/pull/1180/commits/4504499024bbcd79045c2286171554d1e6ff15ba)